axolotl-web/src/store/store.js

```diff
@@ -410,14 +410,14 @@ export default createStore({
         app.config.globalProperties.$socket.send(JSON.stringify(message))
       }
     },
-    openChat(state, chatId) {
-      this.commit("CLEAR_MESSAGELIST");
```

Comment:
If we remove that, we need good testing because I don't remember exactly why I put it there, but it was for a reason. So let's add it to a testing list for the next release.

Reply:
Without this change, I had a bug that the message lists were not loaded.
Before, MessageList.vue dispatched "openChat" and directly after that "getMessageList".
Both of them commited "CLEAR_MESSAGELIST".

I changed "openChat" to dispatch a "getMessageList" and removed one "CLEAR_MESSAGELIST", since once should be enough. Of cause, I'm not entirely sure this won't break anything, but it looked good when I tried it on my phone.